root/static/scripts/edit/URLCleanup.js

```diff
@@ -1,13 +1,26 @@
 /*
+ * @flow
  * Copyright (C) 2010 MetaBrainz Foundation
  *
  * This file is part of MusicBrainz, the open internet music database,
  * and is licensed under the GPL version 2, or (at your option) any
  * later version: http://www.gnu.org/licenses/gpl-2.0.txt
  */
 
+type EntityType =
+ | 'area' | 'artist' | 'event' | 'instrument' | 'label' | 'place'
+ | 'recording' | 'release' | 'release_group' | 'series' | 'work';
+
```

Comment:
I'd suggest to just use `CoreEntityTypeT`. We don't currently have url-url relationships, but we could choose to have them, and we are hoping to add genre-url relationships eventually, so it makes sense to not disallow those either IMO, and it simplifies this.